api.md

```diff
@@ -34,10 +36,12 @@
 * state:element( element_type, data ) - Semi-private, create an element with type and data.
 
 ### Variables
-* state.player - The name of the player.
+* state.player - The name of the current player.
+* state.multiplayer - a list of players connected to the form (on node)
 * state.param - The parameters supplied by form:show.
 * state.def - The form definition.
 * state.is_inv - Boolean which is true if this form is being shown as an inventory.
+* state.nodepos - Node position if attached to node meta
```

Comment:
Any idea for is_inv handling at this place?
```
{
    type = "part_of_players_inventory", --?? or just "inventory" ?
    name = "player name"
}
```
PS: I don't like the type = "singleplayer" because of misleadingly. type="player" is better.